tests/ProspectEventsDataExtension.test.ts

```diff
@@ -0,0 +1,109 @@
+import { ProspectEventsDataExtension } from '../src/manifest/extensions/data/ProspectEventsDataExtension';
+
+describe('ProspectEventsDataExtension', () => {
+  test('only valid type should be acceptable', () => {
+    const ext = getValidExtension();
+    ext.type = 'BANANAS' as any;
+    var issues = ext.validate();
+    expect(issues.length).toBe(1);
+    expect(issues[0]).toBe('Host type is invalid. Value: BANANAS');
+  });
+
+  describe('host', () => {
+    test('host has to be defined', () => {
+      const ext = getValidExtension();
+      delete (ext as any).host;
+      var issues = ext.validate();
+      expect(issues.length).toBe(1);
+      expect(issues[0]).toBe('Host section is missing.');
+    });
+
+    describe('host.icon', () => {
+      test('has to be defined', () => {
+        const ext = getValidExtension();
+        delete (ext.host as any).icon;
+        var issues = ext.validate();
+        expect(issues.length).toBe(1);
+        expect(issues[0]).toBe('Host icon definition is missing');
+      });
+      test('only url should be acceptable', () => {
+        const ext = getValidExtension();
+        ext.host.icon = 'BANANAS';
+        var issues = ext.validate();
+        expect(issues.length).toBe(1);
+        expect(issues[0]).toBe('Host icon definition is invalid. Value: BANANAS');
```

Comment:
I'd extract the strings into variables so that we don't have to repeat them everytime.

Reply:
https://outreach-io.atlassian.net/browse/OREE-927 Created for SDK string localization